llvm/test/Transforms/SampleProfile/pseudo-probe-no-debug-info.ll

```diff
@@ -12,7 +12,7 @@
 
 
 target datalayout = "e-m:e-i64:64-f80:128-n8:16:32:64-S128"
-target triple = "x86_64-unknown-linux-gnu"
+target triple = "x86_64-unknown--"
```

Comment:
Triples with unknown OS default to ELF, so this doesn't really test COFF. 

You'll have to invoke both `opt -mtriple=x86_64` and  `opt -mtriple=x86_64-windows` and omit datalayout/triple in IR.

Reply:
Can we simply remove `target triple` and `target datalayout ` in IR?

Reply:
Yes, they can be omitted in the IR.